ray_julia_jll/gen/README.md

```diff
@@ -0,0 +1,25 @@
+# Building and Publishing JLLs
+
+1. Run the `make.jl` script to build the tarball for the host platform and a given julia-version.
+Builds are required for `linux x86_64` and `macos-arm64` on julia `v1.8.0` and `v1.9.0`.
+It is advised you run this within the python virtual environment associated with the root package.
+Note: rerunning this script _will overwrite_ an existing tarball.
+```
+julia --project -e 'using Pkg; Pkg.develop(path=".."); Pkg.instantiate()'
+julia --project make.jl
+```
+
+2. Once all tarballs have been built for a given host you can publish them as assets to a GitHub pre-release by running the `upload.jl` script, which requires a `GITHUB_TOKEN` environment variable.
+Note: reruning this script will only upload new tarballs and skip any that have already been published.
```

Comment:
Is this still true? Now you can run `upload.jl` after each tarball has been built and you don't need to wait for all of them to complete. In fact this is possibly misleading as you need to run `upload.jl` for each tarball now.

Reply:
You can either
a) run `upload.jl` once all tarballs have been built and (assuming it's a fresh release) all will be uploaded
b) run `upload.jl` after each tarball is built in which case it'll only upload the new one

`ghr` doesn't overwrite existing assets by default - I kept this in place as overwritten assets would be irrecoverable

Reply:
Alright. Based upon these instructions there's an implicit step of collecting all the the tarballs to one machine

Reply:
Not necessarily...you can switch machines and call `make`/`upload` and the new assets will just append the existing release.
For instance, I built and uploaded the two macos assets, then switched machines and repeated the same commands for the linux x86_64 without needing to copy over the macos tarballs.

Reply:
updated the README to indicate you can run once _or_ after each build